qa/download_pip_packages.sh

```diff
@@ -10,13 +10,11 @@ do
     source test.sh
     popd
     echo ${pip_packages}
-    last_config_index=$(python setup_packages.py -n -u $pip_packages --cuda ${CUDA_VERSION})
-    for i in `seq 0 $last_config_index`;
-    do
-        inst=$(python setup_packages.py -i $i -u $pip_packages --cuda ${CUDA_VERSION})
-        if [ -n "$inst" ]
-        then
-            pip download $inst -d /pip-packages
-        fi
+    last_config_index=`./setup_packages.py -n -u $pip_packages --cuda ${CUDA_VERSION}`
+    for i in `seq 0 $last_config_index`; do
+        eval "packages=(`./setup_packages.py -i $i -u $pip_packages --cuda ${CUDA_VERSION} --include-link | tr -d '[],'`)"
+        for pkg in "${packages[@]}"; do
```

Comment:
Why loop?

Reply:
now the output of this script is an array, as opposed to a string